packages/auth/src/core/auth/auth_impl.ts

```diff
@@ -350,16 +353,28 @@ export class AuthImpl implements AuthInternal, _FirebaseService {
     });
   }
 
-  setRecaptchaConfig(config: RecaptchaConfig): void {
-    if (this.tenantId == null) {
-      this._agentRecaptchaConfig = config;
-    } else {
-      this._tenantRecaptchaConfigs[this.tenantId] = config;
-    }
-    if (config.emailPasswordEnabled) {
-      const verifier = new RecaptchaEnterpriseVerifier(this);
-      void verifier.verify();
-    }
+  async initializeRecaptchaConfig(): Promise<void> {
+    await getRecaptchaConfig(this, {
+          clientType: RecaptchaClientType.WEB,
+          version: RecaptchaVersion.ENTERPRISE
+        }).then((response) => {
```

Comment:
Please switch to async/await instead of promise chaining (this is almost there, `response` is the value returned from await getRecaptchaConfig(...) )